src/main/java/com/github/princesslana/eriscasper/rest/auditlog/GetGuildAuditLog.java

```diff
@@ -0,0 +1,37 @@
+package com.github.princesslana.eriscasper.rest.auditlog;
+
+import com.github.princesslana.eriscasper.data.Snowflake;
+import com.github.princesslana.eriscasper.util.QueryStringBuilder;
+import java.util.Optional;
+import org.immutables.value.Value;
+
+/**
+ * @see <a
+ *     href="https://discordapp.com/developers/docs/resources/audit-log#get-guild-audit-log-query-string-parameters">
+ *     https://discordapp.com/developers/docs/resources/audit-log#get-guild-audit-log-query-string-parameters</a>
+ */
+@Value.Immutable
+public interface GetGuildAuditLog {
+  /*
```

Comment:
This just duplicates what is at the linked url in the javadoc?

Reply:
yeap, noticed it late, removed